meetup/views.py

```diff
@@ -43,4 +43,12 @@ def event_by_id(request,event_id):
     guests = event.guest.all()
     return render(request, 'template/event.html', {'event':event,'host':host,'location':location,'guests':guests})
 
-
+@login_required
+def notification(request):
+    notifications = get_all_notifications(request.user)
+    if request.method == 'POST':
+            notifications_to_delete=request.POST.getlist('notification[]')
```

Comment:
Arguably this should go into service layer however since it would just be a simple abstraction not sure if worth it.